apps/src/blockly/googleBlocklyWrapper.ts

```diff
@@ -542,6 +565,10 @@ function initializeBlocklyWrapper(blocklyInstance: GoogleBlocklyInstance) {
     return this.getAllBlocks().filter(block => block.isEnabled());
   };
 
+  extendedWorkspaceSvg.isReadOnly = function () {
```

Comment:
CDO Blockly version: https://github.com/code-dot-org/blockly/blob/v4.1.0/core/ui/block_space/block_space_editor.js#L1181-L1183

Note that in CDO Blockly, we set `this.readOnly_` in the Block Space Editor's constructor logic. We could achieve similar by modifying the workspace after the `inject` call, but that didn't seem warranted for such a simple check. Core Google Blockly frequently checks `workspace.options.readOnly`, so I've chosen to align to that instead.